.circleci/config.yml

```diff
@@ -34,9 +34,7 @@ workflows:
           matrix: &go-version-matrix
             parameters:
               go_version: ["1.17.9"]
-      - indexer_vs_algod_nightly:
-          name: nightly_test_indexer_vs_algod
-          context: slack-secrets
```

Comment:
This is still useful. It is for comparing algod/indexer openapi files.

Reply:
Keeping this shouldn't be too bad, looks like `pytest -sv misc/parity` can be modified to read `algod.oas2.json` by a url instead of the local submodule. 

https://raw.githubusercontent.com/algorand/go-algorand/rel/nightly/daemon/algod/api/algod.oas2.json

Reply:
added this back in the ci and updated algod swagger path in the python script